### DISCUSSION TITLE
Mini.files number column in buffer and support neogit so it works on neogit buffer

### DISCUSSION THREAD

**Discussion by AlejandroSanchez90** [nvim-mini/mini.nvim#1445] [Ideas]
I have two requests for mini files; one is an option to see the number column so I can jump to files quicker, and a fix to when I open it in neogit buffer as it gives an error when I try to open it from there

**Comment by echasnovski** [comment]
> one is an option to see the number column so I can jump to files quicker,

You can enable number column by customizing windows following [these examples](https://github.com/echasnovski/mini.nvim/blob/733afb94ed88d32b839a644dc137604691e46032/doc/mini-files.txt#L386). Here using `MiniFilesWindowUpdate` `User` event is needed. Like this:

```lua
vim.api.nvim_create_autocmd('User', {
  pattern = 'MiniFilesWindowUpdate',
  callback = function(args) vim.wo[args.data.win_id].number = true end,
})
```

> a fix to when I open it in neogit buffer as it gives an error when I try to open it from there

This is expected to be "fixed" on user side by using [appropriate argument for `open()` function](https://github.com/echasnovski/mini.nvim/blob/733afb94ed88d32b839a644dc137604691e46032/doc/mini-files.txt#L703-L717). The error is probably due to executing `MiniFiles.open(vim.api.nvim_buf_get_name(0))` which tries to open file based on Neogit's buffer name, but it is not the actual path on disk.
